workshops/templates/workshops/person.html

```diff
@@ -100,6 +100,17 @@
     {% else %}
     <h5>No lessons.</h5>
     {% endif %}
+
+    {% if languages %}
+    <h5>Languages:</h5>
+    <ul>
+      {% for l in languages %}
```

Comment:
Using `for p in persons` is bad (and we're sinners since we used similar code in AMY), but `for l in languages` is worse (as @gvwilson points out). It should be `for L in languages`, but since `lang` is such a short word it's way better to use it instead of `l` or `L`.